etc/desi.json

```diff
@@ -6,6 +6,17 @@
     },
     "datachallenge":{
         "exclude":[],
+        "dc17a-twopct":{
+            "dc17a-twopct/[^/]+$":"dc17a-twopct/toplevel.tar",
+            "dc17a-twopct/batch/.*$":"dc17a-twopct/batch.tar",
+            "dc17a-twopct/fiberassign/.*$":"dc17a-twopct/fiberassign.tar",
+            "dc17a-twopct/spectro/redux/([0-9a-zA-Z_\\-]+)/exposures/([0-9]){8}/.*$":"dc17a-twopct/spectro/redux/\\1/exposures/\\2.tar",
+            "dc17a-twopct/spectro/redux/([0-9a-zA-Z_\\-]+)/spectra-([0-9]+)/([0-9]+)/.*$":"dc17a-twopct/spectro/redux/\\1/spectra-\\2/\\3.tar",
+            "dc17a-twopct/spectro/redux/([0-9a-zA-Z_\\-]+)/(calib2d|plan|run|zcat|setup).*$":"dc17a-twopct/spectro/redux/\\1/misc.tar",
```

Comment:
After additional review of the code, I'm pretty sure this won't work.  The problem is that htar commands are issued on *one* directory or one set of files in one directory.  The code also makes certain assumptions about the filename of the destination htar file, so you can't just call the destination htar files anything that you want.  I'll run some tests.